_posts/fr/2017-11-22-getting-started-with-ansible.md

```diff
@@ -0,0 +1,144 @@
+---
+layout: post
+title: Commencer avec Ansible, cas concret pour initialiser un VPS
+excerpt: Découvrons Ansible en créant un script qui permet d'initialiser un VPS (appliquer les règles de base + sécurité)
+authors:
+- nkania
+permalink: /fr/getting-start-with-ansible/
+categories:
+    - Non classé
+tags:
+    - linux
+    - vps
+    - serveur
+    - ansible
+image:
+  path: /assets/2017-11-22-getting-started-with-ansible/ansible.png
+---
+
+# Commencer avec Ansible
+
+Quoi de mieux qu'un cas concret pour découvrir et apprendre à utiliser un nouvel outil ? On va donc voir ensemble commencer commencer avec Ansible et l'utiliser afin d'arrêter de s'embêter à répéter à la main les mêmes étapes pour chaque nouveau serveur. Cet article va donc se baser sur un précédent qui vous présentait les bases de la sécurisation d'un nouveau serveur (/fr/securiser-facilement-son-vps-en-quelques-etapes/).
+C'est parti !
+
+### Installation
+
+Bon pour cette partie rien de spécial, je vous renvoie sur la doc afin que vous puissiez installer Ansible sur votre OS : http://docs.ansible.com/ansible/latest/intro_installation.html#installing-the-control-machine
+
+### Configuration
+
+La première chose à faire et d'ajouter l'adresse ip ou hostname de notre serveur dans le fichier hosts de Ansible : 
+
+```bash
+# vim /etc/ansible/hosts
+```
+
+Dans mon exemple j'utilise un VPS chez OVH avec comme distrib ubuntu 16.04, l'installation se fait de base avec juste un user root et une authentification par password, du coup pour tester la connexion on va utiliser la commande suivante :
+
+```bash
+# ansible hostname -m ping -u root --ask-pass
+```
+
+Si comme moi vous recevez l'erreur "/bin/sh: 1: /usr/bin/python: not found\r\n" pas de soucis c'est juste qu'il faut installer le package python-minimal sur son serveur.
```

Comment:
donner la commande pour installer le package manquant

Reply:
formatter l'erreur dans un code block `/bin/sh: 1: /usr/bin/python: not found\r\n`